helper/helper.go

```diff
@@ -296,7 +301,15 @@ func DoCleanup() {
 		logVerbose("Encountered error during cleanup: %v", err)
 	}
 
-	for pipeName, _ := range pipesMap {
+	pipeFiles, _ := filepath.Glob(fmt.Sprintf("%s_[0-9]*", *pipeFile))
+	for _, pipeName := range pipeFiles {
+		if !wasSigpiped {
+			logVerbose("Opening/closing pipe %s", pipeName)
+			err = openClosePipe(pipeName)
```

Comment:
Please add a comment why the open/close is needed here

Reply:
added